tests/core/test_axes.py

```diff
@@ -131,6 +131,17 @@ def test_add_perpendicular_axes(simulation_builder, persons):
     assert simulation_builder.get_input('salary', '2018-11') == approx([0, 1500, 3000, 0, 1500, 3000])
     assert simulation_builder.get_input('pension', '2018-11') == approx([0, 0, 0, 2000, 2000, 2000])
 
+
+def test_axis_on_existing_variable(simulation_builder, persons):
```

Comment:
Is it just on an existing variable or in a variable with an already defined input for that period?

Reply:
Found my answer tinkering with the code: it fails when there is already and existing input for a variable of the requested period.

The reason it was failing is sadly not the good one: if we were to give as an expansion count `1`, it wont expand at all —which makes sense, but maybe it is not the least surprising behaviour.